homesensorhub/sensors/types/light.py

```diff
@@ -0,0 +1,24 @@
+"""Module which contains the light type class."""
+from routing.payload import Payload
+from sensors.types.sensor_type import SensorType
+
+import datetime
+
+
+class Light(SensorType):
+    """Class which represents a type of light value collected by light sensor modules."""
+
+    TYPE = 'Light'
+
+    def get_payload(self) -> Payload:
+        """
+        Collect payload for this sensor type.
+
+        Trigger a data collection for this type.
+        """
+        payload = Payload(self.TYPE,
```

Comment:
move method to SensorType parent class

Reply:
Based on this commend I ended up changing the structure again. Ty. :+1: